packages/app/src/settings/SettingsContainer.vue

```diff
@@ -43,7 +43,7 @@
       variant="outline"
```

Comment:
Above for the `t('settingsPage.footer.text')` should be modified to include the testing type since it's currently wired to only show component. Something like:
```
const footerText = computed(() => t('settingsPage.footer.text', { testingType: props.gql.currentProject?.currentTestingType === 'component' ? 'component' : 'E2E' }))
```

Then you can render `{{ footerText }}`

Reply:
Great callout!